server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -263,6 +265,21 @@ export class DeliLambda extends EventEmitter implements IPartitionLambda {
                     this.setNoopConsolidationTimer();
                     continue;
                 }
+
+                // Check if Deli is over the max ops since last summary nack limit
+                if (this.serviceConfiguration.deli.summaryNackMessages.enable && !this.nackMessages) {
+                    const opsSinceLastSummary = this.sequenceNumber - this.durableSequenceNumber;
+                    if (opsSinceLastSummary > this.serviceConfiguration.deli.summaryNackMessages.maxOps) {
+                        // this op brings us over the limit
+                        // start nacking non-system ops and ops that are submitted by non-summarizers
+                        this.nackMessages = {
```

Comment:
If this.nackMessages is not undefined at the time of session end (lambda close), we should probably fail the session end metric in logSessionEndMetrics